### DISCUSSION TITLE
Problem. How to? Docker + Fail2ban + Authelia 🤷🏻‍♂️ [SOLVED]

### DISCUSSION THREAD

**Discussion by taimadoCE** [fail2ban/fail2ban#3395] [Q&A] [answered]
Hi, friends. I'm trying to install Fail2ban into docker for monitoring the logs of the container of Authelia.
The main problem I found is that I don't know to mount in local the logs of Authelia...Where are they?

![imagen](https://user-images.githubusercontent.com/57089805/198874298-9296a528-1154-49b4-98b2-68a8497ec153.png)

I'm reading the section of logs in the web of Authelia.
![imagen](https://user-images.githubusercontent.com/57089805/198874414-197d6f6a-d933-49b8-8c52-1e89f6e2bbcf.png)

I get the log output as you can see:
![imagen](https://user-images.githubusercontent.com/57089805/198874445-19d8d731-54a8-4ec3-894e-977d4c1fd3c2.png)

But, what's next? How do I configure the docker compose of Fail2ban? 
I think that I do something wrong, sure, but...

**Comment by sebres** [comment]
See https://github.com/crazy-max/docker-fail2ban

Maybe @crazy-max could help.

**Reply by taimadoCE** [reply]
I've read before. The main problem is that I don't know how to read the authelia.log from the fail2ban container. I get the log in the Authelia folder but...How can I access to it from the Fail2ban?
![imagen](https://user-images.githubusercontent.com/57089805/198895489-f7ebc909-ac08-44fa-9993-917a786290ec.png)

**Reply by sebres** [reply]
I have no idea what is Authelia and where it basically does its logging, but...
1. If it is running in container (and it is not the same container as fail2ban) you'd need to mount its volume with log-file to the container where fail2ban is running, see https://stackoverflow.com/a/43560093
2. If it is running on host (and fail2ban in container), see https://stackoverflow.com/a/23455537
3. also see https://www.digitalocean.com/community/tutorials/how-to-share-data-between-the-docker-container-and-the-host
4. also see [[if ](https://stackoverflow.com/questions/71591590/how-to-forward-systemd-logs-from-a-docker-container-to-the-host-journalctl)](https://projectatomic.io/blog/2016/10/playing-with-docker-logging/)

Anyway I don't think someone would help you here, since it is fail2ban repository and your question is rather: 
how I can provide some log-file or journal from one docker container to another (or to host)... 
and it is docker (containerization) question either...

**Comment by taimadoCE** [comment, marked as answer]
The solution here:

https://github.com/authelia/authelia/discussions/4300#discussioncomment-4013978
